docs/animations/Blur.md

```diff
@@ -1,74 +1,86 @@
 # Blur
 
-The **Blur animation behavior** selectively blurs a XAML element by increasing or decreasing pixel size.
-Sometimes you want an element to appear slightly out of focus, but to be familiar to the user from other locations within an app.  Rather than having to rasterize the XAML into an image and apply a blur, you can apply a BlurBehavior to the original element at run time. 
-
-**NOTE**:  This animation REQUIRES the [Windows 10 Anniversary SDK 14393](https://blogs.windows.com/windowsexperience/2016/07/18/build14393/) 10.0.14393.0 in order for it to work.
+The Blur animation blurs a XAML element by increasing or decreasing pixel size. Blur animation is applied to all the XAML elements in its parent control/panel. Blur animation doesn't affect the functionality of the control.
 
 ## Syntax
 
-You can either use the blur behavior from your XAML code:
+**XAML**
 
 ```xml
+<Page ...
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"  
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors"/>
 
-    <interactivity:Interaction.Behaviors>
+<interactivity:Interaction.Behaviors>
     <behaviors:Blur x:Name="BlurBehavior" 
-           Value="10" 
-           Duration="500" 
-           Delay="250" 
-           AutomaticallyStart="True"/>
-    </interactivity:Interaction.Behaviors>
-
+            Value="10" 
+            Duration="2500" 
+            Delay="250" 
+            AutomaticallyStart="True"/>
+</interactivity:Interaction.Behaviors>
 ```
 
-or directly from code:
+**C#**
 
 ```csharp
+MyUIElement.Blur(value: 5, duration: 2500, delay: 250).Start();
+await MyUIElement.Blur(value: 5, duration: 2500, delay: 250).StartAsync();  //Blur animation can be awaited
+```
 
-ToolkitLogo.Blur(value: 10, duration: 500, delay: 250);       
+## Sample Output
 
-```
+![Blur Behavior animation](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/Blur/Sample-Output.gif)
 
 ## Properties
```

Comment:
Ah. Not yet, still working with the docs team to process our nugets and not sure what the format will be